### DISCUSSION TITLE
Unknown temperatures and fan speeds

### DISCUSSION THREAD

**Discussion by OlwinFroon** [timmo001/system-bridge#3327] [Help]
I wanted to use System Bridge for rudimentary health monitoring: main HDD/SSD usage in percent, CPU and GPU temperature, fan speeds if possible.

I run a Windows PC with Intel Core i5 12500 CPU and Intel UHD Graphics 770.

The HWMonitor app shows all kinds of fan speeds and temperatures.

System Bridge lists a whole lot of sensors, too, but very few of the fan and temperature values I'd actually want:
- GPU fan speed: unknown
- GPU temperature: unknown
- CPU temperature: unknown

I checked all the JSON output tabs as well, there's nothing.

So I guess the windows_sensors module isn't up to this, yet.
What can I do?

**Comment by VidFerris** [comment]
I had this problem and after digging through the issues history I resolved it by launching System Bridge as administrator - this was discussed a very long time ago at https://github.com/timmo001/system-bridge/discussions/635. @timmo001 might be a good idea to suggest launching as admin in Windows to the FAQ for if this happens?

**Reply by OlwinFroon** [reply]
Duh!
Thanks so much for this!
HDD usage worked before, CPU temp and fans show up now.
That'll be enough for my use case.

GPU, however, still reports memory_used and power_usage only, everything else (especially fan_speed and temperature) are NULL.
I guess I can live with that.

Thanks again, you made my day!
